spec/controllers/downloads_controller_spec.rb

```diff
@@ -44,14 +44,19 @@
 
       context "with an alternative file" do
         context "that is persisted" do
+          let(:file) { File.open(fixture_file_path('world.png'), 'rb') }
+
+          # rubocop:disable Style/Semicolon
```

Comment:
Do you object to the following?

```ruby
let(:content) do
  file.rewind
  file.read
end
```

My preference is to discourage working around Rubocop unless there are reasons to do so. (If you have reasons, I'm happy to hear them.)